### ISSUE TITLE
Chrome Preferences not passed to Selenium Driver

### ISSUE THREAD

**Issue by bgsmalls** [cobrateam/splinter#382]
After trying to disable images using splinter, I discovered that the options are not properly passed to the Selenium Driver. Using the Selenium Chrome Driver works without an issue.

Specifically the experemental options are treated in an non-standard way by chrome.
Fortunatelly the issue is easy to fix.

File: /usr/local/lib/python2.7/dist-packages/splinter/driver/webdriver/chrome.py

OLD-with-Bug, line 17-20:

```
def __init__(self, user_agent=None, wait_time=2, fullscreen=False,
                 **kwargs):

        options = Options()
```

NEW-Bugfix, line 17-20:

```
def __init__(self, user_agent=None, wait_time=2, fullscreen=False, options=None,
                 **kwargs):

        options = options or Options()
```

Could you please add this fix to the main development ? Thanks a lot.

**Comment by coanor** [comment]
We really need such a option to passed to `splinter.Browser()`,  one of mine is `--disable-setuid-sandbox`, see stack overflow [thread](http://stackoverflow.com/questions/37696854/splinter-how-to-add-chrome-options).

**Comment by bmcculley** [comment]
I have introduced the passing of options as a list in my [dev branch](https://github.com/bmcculley/splinter/tree/dev) if you would like to have a look.

For example:

 ```
from splinter import Browser

options = ['--start-maximized', '--disable-setuid-sandbox']
browser = Browser('chrome', chrome_options=options)
```

**Comment by bmcculley** [comment]
This is addressed in pull request #486
